### PR TITLE
NFD Operator known issue is resolved in latest z-stream

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

**Review by bergerhoffer — COMMENTED** [review]
This LGTM, but more just a question of general practices - is it normal to remove known issues from release notes once they have been resolved? I wasn't sure whether it needed to be kept there for users that aren't on that z-stream release yet. Or if the resolution of the BZ should be in the RN for the particular z-stream that fixed it.

**Comment by codyhoag** [comment]
@bergerhoffer we don't typically track known issues, so they usually stay in. For this case, it was a pretty high-level blocker issue that was requested to be removed when it was fixed. It involved missing Operators and the workaround was to pull them manually from GitHub, which isn't the best look. I don't believe there was a doc text for the fix, so nothing was documented in the z-stream. However, I do think it makes sense to call it out since we're removing it.

I've updated the PR to include its bug fix description in the latest z-stream. Can you take a second look?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2541,3 +2514,14 @@ Issued: 2020-11-09
 Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
 
 link:https://access.redhat.com/solutions/5551151[{product-title} 4.6.3 container image list]
+
+[id="ocp-4-6-3-bug-fixes"]
+==== Bug fixes
+
+* Due to a known issue, the GPU Operator and Node Feature Discovery (NFD) Operator were not available on a fresh installation of {product-title} 4.6.1. You were required to install {product-title} 4.5 and upgrade the cluster to version 4.6.1 to use the GPU and NFD Operators. This issue has been fixed, and the GPU and NFD Operators are now available on a fresh installation of {product-title} 4.6.3 and later. link:https://bugzilla.redhat.com/show_bug.cgi?id=1890673[*BZ#1890673*].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-6-release-notes.adoc:2521]: I don't think there's normally a period after the BZ link

**Review by bergerhoffer — COMMENTED** [review]
Thanks for the explanation @codyhoag! I do feel better about the known issue being removed if it is mentioned as being fixed in the z-stream. Thanks for looking into it!

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (codyhoag) [reply release_notes/ocp-4-6-release-notes.adoc:2521]: Ah, thank you :sweat_smile:

**Pull request merged**